### ISSUE TITLE
Support for GitHub Enterprise

### ISSUE THREAD

**Issue by johnrc** [Financial-Times/github-label-sync#13]
I would like to add a configuration option to use GitHub Enterprise instead of GitHub. Adding support is trivial since you're using `octonode` by passing in `hostname` as an option during initialization. For example:

```js
class ApiClient {

	constructor (accessToken) {
		this.apiClient = github.client(accessToken, { hostname: 'ghe.example.com/api/v3' });
	}
...
}
```

How would you like this option passed in? (e.g. `--hostname` option, `GITHUB_HOSTNAME` environment variable, etc.)

**Comment by rowanmanning** [comment]
Hey @johnrc, yep would deffo accept a PR and your suggestion of `--hostname` + `GITHUB_HOSTNAME` is great

**Comment by JakeChampion** [comment]
This has been resolved via #20

**Comment by johnrc** [comment]
Sorry for never responding until now. We were required to move from GH issues to Jira and I forgot about this. I'm glad someone picked it up.
